diff --git a/src/OpenCvSharpExtern/include_opencv.h b/src/OpenCvSharpExtern/include_opencv.h
index 77abc1fc..ddbf4e21 100644
--- a/src/OpenCvSharpExtern/include_opencv.h
+++ b/src/OpenCvSharpExtern/include_opencv.h
@@ -39,9 +39,6 @@
 #include <opencv2/highgui/highgui_winrt.hpp>
 #endif
 #include <opencv2/core/utils/logger.hpp>
-#ifndef NO_HIGHGUI
-#include <opencv2/highgui/highgui_c.h>
-#endif
 #include <opencv2/imgproc/imgproc_c.h>

 #ifndef NO_VIDEO
