diff --git a/src/OpenCvSharpExtern/include_opencv.h b/src/OpenCvSharpExtern/include_opencv.h
index 77abc1fc..e0f3e560 100644
--- a/src/OpenCvSharpExtern/include_opencv.h
+++ b/src/OpenCvSharpExtern/include_opencv.h
@@ -39,51 +39,8 @@
 #include <opencv2/highgui/highgui_winrt.hpp>
 #endif
 #include <opencv2/core/utils/logger.hpp>
-#ifndef NO_HIGHGUI
-#include <opencv2/highgui/highgui_c.h>
-#endif
 #include <opencv2/imgproc/imgproc_c.h>
 
-#ifndef NO_VIDEO
-#include <opencv2/video.hpp>
-#endif
-
-#ifndef NO_STITCHING
-#include <opencv2/stitching.hpp>
-#endif
-
-#ifndef NO_ML
-#include <opencv2/ml.hpp>
-#endif
-
-#ifndef NO_CONTRIB
-#include <opencv2/shape.hpp>
-#ifndef _WINRT_DLL
-#include <opencv2/superres.hpp>
-#include <opencv2/superres/optical_flow.hpp>
-#endif 
-// opencv_contrib
-#include <opencv2/aruco.hpp>
-#include <opencv2/aruco/charuco.hpp>
-#include <opencv2/bgsegm.hpp>
-#include <opencv2/img_hash.hpp>
-#include <opencv2/line_descriptor.hpp>
-#include <opencv2/optflow.hpp>
-#include <opencv2/quality.hpp>
-#include <opencv2/tracking.hpp>
-#include <opencv2/xfeatures2d.hpp>
-#include <opencv2/ximgproc.hpp>
-#include <opencv2/saliency.hpp>
-#include <opencv2/xphoto.hpp>
-#ifndef _WINRT_DLL
-#include <opencv2/wechat_qrcode.hpp>
-#include <opencv2/dnn.hpp>
-#include <opencv2/dnn_superres.hpp>
-#include <opencv2/face.hpp>
-#include <opencv2/text.hpp>
-#endif
-#endif // NO_CONTRIB
-
 #include <vector>
 #include <algorithm>
 #include <iterator>
